activerecord/test/cases/migration_test.rb

```diff
@@ -193,6 +193,15 @@ def test_create_table_with_if_not_exists_true
         t.string :foo
       end
     end
+
+    # Do not drop table if existence guard option is present
+    assert_nothing_raised do
```

Comment:
Can we extract this to its own test? Maybe `test_create_table_with_if_not_exists_true_and_force_true`